keanu-python/keanu/generated/vertex.py

```diff
@@ -0,0 +1,54 @@
+## This is a generated file. DO NOT EDIT.
+
+from py4j.java_gateway import java_import
+from keanu.base import KeanuContext, Vertex
+
+k = KeanuContext().jvm_view()
+
+
+java_import(k, "io.improbable.keanu.vertices.bool.nonprobabilistic.operators.binary.compare.GreaterThanVertex")
```

Comment:
Is it possible in our `vertex.py.ftl` to do two iterations of the loop? One for the imports and one for the constructors. It'll just look so much clearer if all the imports are at the top then all the constructors after it